Index: 0x00-python-hello_world/7-edges.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/0x00-python-hello_world/7-edges.py b/0x00-python-hello_world/7-edges.py
new file mode 100644
--- /dev/null	(date 1654037419490)
+++ b/0x00-python-hello_world/7-edges.py	(date 1654037419490)
@@ -0,0 +1,8 @@
+#!/usr/bin/python3
+word = "Holberton"
+word_first_3 = word[:3]
+word_last_2 = word[-2:]
+middle_word = word[1:-1]
+print("First 3 letters: {}".format(word_first_3))
+print("Last 2 letters: {}".format(word_last_2))
+print("Middle word: {}".format(middle_word))
